app/grandchallenge/products/utils.py

```diff
@@ -78,10 +79,9 @@ def _create_company(self, row):
         c.email = row["Email address (public)"]
         c.description = row["Company description"]
         c.description_short = self._split(row["Company description"], 200)
-        image_name = row["Company name"]
-        for ch in [" ", ".", "-", ","]:
-            image_name = image_name.replace(ch, "")
-        img_file = self.images_path.glob(f"**/logo/{image_name.lower()}.*")
+        slug = re.sub(r"[\W_]+", "", row["Company name"]).lower()
```

Comment:
There is a nice django function to slugify a title: https://docs.djangoproject.com/en/3.0/ref/utils/#django.utils.text.slugify